src/cr/cube/stripe.py

```diff
@@ -19,15 +19,28 @@
 
 
 class TransformedStripe(object):
-    """Stripe reflecting application of all transforms."""
+    """Stripe reflecting application of all transforms.
```

Comment:
I might be wrong in my gut feeling, but I feel a need for a comma after the word "stripe". Either that or a more redundant construction "that reflects", or some similar indirection.